tests/engines/collections/acceptance/submit/submit-test.ts

```diff
@@ -30,33 +33,164 @@ module('Collections | Acceptance | submit', hooks => {
             primaryCollection,
             licensesAcceptable,
         });
+
+        const store = this.owner.lookup('service:store');
+
+        const newTitle = 'New Title';
+        const newDescription = 'New description.';
+
         await visit(`/collections/${provider.id}/submit`);
+
+        /* Select a project */
+
         await percySnapshot('Collections | Acceptance | submit | select project');
+
+        // open item picker
         await untrackedClick('[data-test-collections-item-picker] .ember-power-select-trigger');
-        await untrackedClick(`[data-test-collections-node-title="${nodeToBeAdded.title}"]`);
+        // select node
+        const nodeToBeAddedOption = document.querySelector(
+            `[data-test-collections-node-title="${nodeToBeAdded.title}"]`,
+        );
+        if (nodeToBeAddedOption) {
+            await untrackedClick(nodeToBeAddedOption);
```

Comment:
Could a comment here on when/why this condition would be false be added? I wasn't able to glean why this condition is needed here and am wary of conditionals built into tests.

Reply:
The types force you to do this. The type of `document.querySelector()` is `Element | null`. `untrcakedClick()` expects an `Element` so we have to use a condition to narrow the type. But it's also a good idea because there is no guarantee that it will find the element (as they types denote)!